microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/security/ListenerSecurityConfigurationTest.java

```diff
@@ -84,30 +85,27 @@ public static WebArchive createServicesProviderDeployment() {
                         OpenApiModelReader.class,
                         OpenApiFilter.class)
                 .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
-        return deployment;
     }
 
     static class OpenApiExtensionSetup implements ServerSetupTask {
 
         @Override
         public void setup(ManagementClient managementClient, String containerId) throws Exception {
             jbossHome = arquillianContainerProperties.getContainerProperty("jboss", "jbossHome");
-            onlineManagementClient = ManagementClientProvider.onlineStandalone();
+            listenersConfigurationOnlineManagementClient = ManagementClientProvider.onlineStandalone();
```

Comment:
Could be `listenersConfigurationOnlineManagementClient` initialized in `@BeforeClass`, please?

Reply:
Yes, sure, sorry about this, I definitely missed it.

Reply:
Done